webapp/package.json

```diff
@@ -3,60 +3,47 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@material-ui/core": "^4.12.3",
-    "@monaco-editor/react": "^4.3.1",
-    "@patternfly/patternfly": "^4.115.2",
-    "@patternfly/react-core": "^4.216.1",
-    "@patternfly/react-icons": "^4.67.1",
-    "@patternfly/react-table": "^4.85.1",
-    "@types/history": "^4.7.9",
-    "@types/jsonpath": "^0.2.0",
-    "@types/keycloak-js": "^3.4.1",
-    "@types/luxon": "^1.27.1",
-    "@types/react": "^17.0.15",
-    "@types/react-autosuggest": "^10.1.5",
-    "@types/react-dom": "^17.0.9",
-    "@types/react-redux": "^7.1.18",
-    "@types/react-router-dom": "^5.1.8",
-    "@types/react-table": "^7.7.2",
-    "@types/recharts": "^2.0.0",
-    "@types/redux": "^3.6.0",
-    "@types/redux-thunk": "^2.1.0",
-    "@types/uuid": "^8.3.4",
-    "axios": "^0.21.2",
-    "clsx": "^1.1.1",
-    "codemirror": "^5.62.2",
-    "connected-react-router": "^6.9.1",
-    "fetchival": "^0.3.3",
-    "github-markdown-css": "^5.1.0",
-    "history": "^4.10.1",
-    "immutable": "^4.0.0-rc.14",
-    "js-beautify": "^1.14.1",
-    "jshint": "^2.13.0",
-    "jsonpath": "^1.1.1",
-    "keycloak-js": "^19.0.2",
-    "luxon": "^2.0.1",
-    "moment": "^2.29.4",
-    "monaco-editor": "^0.25.2",
-    "react": "^17.0.2",
-    "react-autosuggest": "^10.1.0",
-    "react-dom": "^17.0.2",
-    "react-helmet": "^6.1.0",
-    "react-icons": "^4.2.0",
-    "react-markdown": "^7.1.1",
-    "react-monaco-editor": "^0.43.0",
-    "react-redux": "^7.2.4",
-    "react-router": "^5.2.0",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "^4.0.3",
-    "react-table": "^7.7.0",
-    "react-to-print": "^2.13.0",
-    "recharts": "^2.1.13",
-    "redux": "^4.1.0",
-    "redux-thunk": "^2.3.0",
-    "reselect": "^4.0.0",
-    "typescript": "^4.3.5",
-    "uuid": "^8.3.2"
+    "@monaco-editor/react": "4.4.6",
+    "@patternfly/patternfly": "4.224.2",
+    "@patternfly/react-core": "4.276.6",
+    "@patternfly/react-table": "4.112.39",
+    "@types/history": "4.7.11",
+    "@types/jsonpath": "0.2.0",
+    "@types/luxon": "3.2.0",
+    "@types/react": "17.0.37",
+    "@types/react-autosuggest": "10.1.5",
+    "@types/react-dom": "17.0.11",
+    "@types/react-redux": "7.1.20",
+    "@types/react-router-dom": "5.3.3",
```

Comment:
```suggestion
    "@types/react-router-dom": "6.9.0",
```

Reply:
can we go with above suggested version?

Reply:
This package is the Typescript definitions for the `react-router-dom` package. I assume the versions need to be aligned between `@types/react-router-dom` and `react-router-dom` otherwise we will have inconsistencies between the Typescript types and the API.  I think we can update this when we update the router/history/redux-connector versions

Reply:
okay